packages/wrapped-keys/src/lib/lit-actions-client/sign-transaction.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  AccessControlConditions,
+  ILitNodeClient,
+  SessionSigsMap,
+} from '@lit-protocol/types';
+
+import { postLitActionValidation } from './utils';
+import {
+  EthereumLitTransaction,
+  SerializedTransaction,
+  StoredKeyMetadata,
+} from '../types';
+
+interface SignTransactionWithLitActionParams {
+  litNodeClient: ILitNodeClient;
+  pkpSessionSigs: SessionSigsMap;
+  litActionIpfsCid: string;
+  unsignedTransaction: EthereumLitTransaction | SerializedTransaction;
+  storedKeyMetadata: StoredKeyMetadata;
+  accessControlConditions: AccessControlConditions;
+  broadcast: boolean;
+}
+
+export async function signTransactionWithLitAction({
+  accessControlConditions,
+  broadcast,
+  litActionIpfsCid,
+  litNodeClient,
+  pkpSessionSigs,
+  storedKeyMetadata: { ciphertext, dataToEncryptHash, pkpAddress },
```

Comment:
Why do it like this?

Reply:
Can you elaborate on what you mean?

Reply:
Ig the idea is to club the DB data in a single var